manual/azure-data-studio-sql-server-admin-pack/README.md

```diff
@@ -0,0 +1,20 @@
+# <img src="https://cdn.rawgit.com/bbtsoftware/chocolatey-packages/19e7388f56f2faf2e2b7a270984d51e93ff8a1cb/icons/vscode-cake.png" width="48" height="48"/> [vscode-cake](https://chocolatey.org/packages/vscode-cake)
+
+Admin Pack for SQL Server is a collection of popular database administration extensions to help you manage SQL Server.
+
+## Notes
+
+When you download this pack, you will download the following extensions:
+
+- [SQL Server Agent](https://docs.microsoft.com/sql/azure-data-studio/sql-server-agent-extension?view=sql-server-2017)
+  - List SQL Server Agent Jobs configured on a SQL Server
+  - View Job History with job execution results
+  - Basic Job Control to start and stop jobs
+- [SQL Server Profiler](https://docs.microsoft.com/sql/azure-data-studio/sql-server-profiler-extension?view=sql-server-2017)
+  - Browse through extended events and view associated T-SQL
+  - View and manage sessions
+  - Filter search of events
+- [SQL Server Import](https://docs.microsoft.com/sql/azure-data-studio/sql-server-import-extension?view=sql-server-2017)
+  - Use the Import Flat File Wizard to import .csv and .txt files to a SQL table using AI-based framework to simplify experience
+- [SQL Server dacpac](https://docs.microsoft.com/sql/azure-data-studio/sql-server-dacpac-extension?view=sql-server-2017)
+  - Use the Data-Tier Application Wizard to deploy and extract .dacpac files and import and export .bacpac files
```

Comment:
Add a note that this package always installs the latest version of the extension (like done [here](https://github.com/bbtsoftware/chocolatey-packages/blob/master/manual/vscode-cake/vscode-cake.nuspec#L25-L26))